sample_app_dagger/src/main/java/com/chumarin/stanislav/sample_app_dagger/di/AppModule.java

```diff
@@ -0,0 +1,71 @@
+package com.chumarin.stanislav.sample_app_dagger.di;
+
+import androidx.annotation.NonNull;
+
+import com.chumarin.stanislav.sample_app_dagger.App;
+import com.chumarin.stanislav.sample_app_dagger.Session;
+import com.chumarin.stanislav.sample_app_dagger.SessionTrackerListener;
+import com.chumarin.stanislav.sample_app_dagger.SessionTrackerStorage;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import javax.inject.Singleton;
+
+import dagger.Binds;
+import dagger.Module;
+import dagger.Provides;
+import vit.khudenko.android.sessiontracker.ISessionStateTransitionsSupplier;
+import vit.khudenko.android.sessiontracker.ISessionTrackerStorage;
+import vit.khudenko.android.sessiontracker.SessionTracker;
+import vit.khudenko.android.sessiontracker.Transition;
+
+@Module
+class AppModule {
+
+//    @Provides
```

Comment:
If the commented code is not required indeed, then please remove it.

Reply:
Removed commented code